### PR TITLE
Karate Race 6S polishing

### PR DESCRIPTION
**Pull request by limonspb**
1. Added experimental BOSH option
2. Reduced AG gain to 3.0
3. Adding corresponding rc_smoothing to the RC_LINK options
4. Replaced limon's spicy D gain with Spicy TPA
5. Added options_group for RC_LINKS for better look
6. Added indentation to the code

Original Karate PR:
https://github.com/betaflight/firmware-presets/pull/41

### CODE REVIEW

**Comment by sugaarK** [comment]
we need a logo let Me speak to @KarateBrot

**Comment by limonspb** [comment]
> we need a logo let Me speak to @KarateBrot

logo in the next PR lol

**Comment by sugaarK** [comment]
lols just the logo it doesn't need the karate!!! needs a roundel though cause it'll look funny in dark mode


I think that's perfect though

**Comment by limonspb** [comment]
> lols just the logo it doesn't need the karate!!! needs a roundel though cause it'll look funny in dark mode
> 
> I think that's perfect though

lets review this PR without logo plz.

**Review by sugaarK — COMMENTED** [review]
oops

File: presets/4.3/tune/karate_race.txt [hunk 1]
```diff
@@ -26,6 +26,11 @@ set gyro_lpf1_dyn_min_hz = 500
 set gyro_lpf1_dyn_max_hz = 1000
 set simplified_gyro_filter = OFF
 
+#$ OPTION BEGIN (UNCHECKED): Bosh (BMI) gyro - experimantal
```
Comment (sugaarK) [inline presets/4.3/tune/karate_race.txt:29]: shouldn't this have the OSR4 cli command just to be sure... also ive had a number of people contact me and say they couldn't get the Bosch flying good so my solution was to kick the fixed stage2 gyro back to 250hz..

Comment (sugaarK) [reply presets/4.3/tune/karate_race.txt:29]: I don't think all these Fcs are equal.. its the usual issue im seeing though not enough hardware filtering

**Review by sugaarK — APPROVED** [review]

File: presets/4.3/tune/karate_race.txt (hunk 1 above)
Comment (limonspb) [reply presets/4.3/tune/karate_race.txt:29]: > I don't think all these Fcs are equal.. its the usual issue im seeing though not enough hardware filtering

so there are FCs with BMI gyro + OSR4 mode that requires MORE filtering that MPU6000? That's pretty wild for OSR4 mode.

**Comment by limonspb** [comment]
added logo as per @KarateBrot but moved it to the right. Square logos accupy too much room in the middle shifting the description too low :)

image

**Review by KarateBrot — APPROVED** [review]

**Comment by dezignhero** [comment]
@sugaarK @limonspb Following up from a discussion on Slack.  Would it be possible to add a "safe" range for the Dynamic Idle setting?  Since it's written as 2000kv I assume I need to have 2000kv exactly, but I've been told it's actually usable to for 1800kv-2200kv range which is quite broad.  Therefore could we change `#dynamic idle for 6s 2000kv` to `#dynamic idle for 6s 1800-2200kv` ?

**Pull request merged**